spec/controllers/availabilities_controller_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe AvailabilitiesController, type: :controller do
+  describe 'GET availabilities#index' do
+    before do
+      create_list(:time_range, 10)
+      @user = create(:user)
+      sign_in @user, scope: :user
+      get :index
+    end
+    it { should respond_with :ok }
+
+    it 'creates availability' do
+      TimeRange.all.to_a.all? do |time_range|
+        Availability.exists?(user: @user, time_range: time_range)
+      end
+    end
+  end
+
+  describe 'POST availabilities#update_availabilities' do
+    before do
+      @user = create(:user)
+      sign_in @user, scope: :user
+      create_list(:time_range, 10)
+      get :index
+    end
+
+    it 'redirects to availabilities_path' do
+      post :update_availabilities, params: { availability_ids: [] }
+      should redirect_to availabilities_path
+    end
+
+    it 'should not create new availability' do
```

Comment:
Should not create new availability when...?

Reply:
in all situation

Reply:
Isn't it only when `availabilitiy_ids` is empty? I think better to expand out `should not create new availability when availability_ids is empty`

Reply:
no, because the availabilities already exists, the update should only update between false & true(?)

Reply:
Ooooh omg okay I just get what you mean